app.js

```diff
@@ -28,7 +28,7 @@ app.use((err, request, response, next) =>{
     response.status(404).send(err.msg)
```

Comment:
we could make this more dynamic and instead of saying if err.status===404 and just say if err.status and err.msg are true.

Then we can send err.status and err.msg back to the client